piezo_web_app/PiezoWebApp/src/utils/route_helper.py

```diff
@@ -19,3 +21,20 @@ def is_scheme_secure(scheme):
     if scheme.lower() == 'ftp':
         return False
     raise ValueError(f'"{scheme}" not recognised as a valid scheme')
+
+
+def is_valid_pod_name(label):
```

Comment:
Would it be possible to change "label" to "name" or equivalent? It could get confusing with the regex checks for the label argument